server/src/immich/api-v1/asset/asset.service.ts

```diff
@@ -338,7 +340,7 @@ export class AssetService {
       fileModifiedAt: dto.fileModifiedAt,
       localDateTime: dto.fileCreatedAt,
 
-      type: mimeTypes.assetType(file.originalPath),
```

Comment:
We can leave this as is, it is self-explanatory so you don't need to add a comment

Reply:
Actioned